README.md

```diff
@@ -0,0 +1,118 @@
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
+
+* System dependencies
+
+* Configuration
+
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
+* ...
+# Enurable
+
+A project to create the Ruby enumerable module.
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+## Clone this repository
+
+  $ git@github.com:NduatiKagiri/enumerable.git
+  $ cd enumerable
+
+## Run Project
+  $ Kali Rolling WSL
+
+### Install
+  $ rbenv install 3.0.1
+  $ rbenv global 3.0.1
+  $ ruby -v
```

Comment:
- [ ] The formatting here is inconsistent, resulting on weird URLs shown on your readme which are confusing because these are console commands.

In order to correctly format code or commands on your readme, you can use this syntax:

![image](https://user-images.githubusercontent.com/57421823/203853280-73316c25-c9bb-4f54-99f8-2a11bd2098c9.png)

Resulting in this perfectly formatted code:

```bash
  $ rbenv install 3.0.1
  $ rbenv global 3.0.1
  $ ruby -v
```
"bash" is the language here, because these are console commands. 💪